### DISCUSSION TITLE
How to track investment certificates?

### DISCUSSION THREAD

**Discussion by MarcoCaneve** [ghostfolio/ghostfolio#2024] [Q&A]
Hello, 

I need to track some investment certificates. They behave similarly to normal stocks; you can buy and sell them in the market, and they pay dividends. The problem is that these products are not tracked by most data sources (e.g., Yahoo Finance), so I cannot add them as I would with a normal stock.

I only need to track the performance based on dividends and capital gains (by manually enter the values), not the daily price. 
How can I add them to the software?

**Comment by Wibbbs** [comment]
I am looking for the same thing. Even when I add them as a valuable they don’t get included in the total overview.
